### ISSUE TITLE
Accessibility: Incorrect control type is defined for all menus like 'File and Edit and View and Tools...' in MenuStrip

### ISSUE THREAD

**Issue by vladimir-krestov** [dotnet/winforms#1328]
* .NET Core Version: 3.0.1-preview8-012981
* Have you experienced this same bug with .NET Framework?: Yes

**Problem description:**
*User Experience:*  
User who depends on screen reader won't be able to get the correct purpose of the controls on tool, if Incorrect control type is defined for all menus like 'File and Edit and View and Tools...' on SSMA.

**Repro steps:**
1. Create new .Net Core app
2. Add MenuStrip control to Form
3. Add some items to MenuStrip
4. Open the Inspect tool and look on control type of items of the first MenuStrip layer

**Actual behavior:** 
Incorrect control type 'Menu item' is defined for all menus like 'File and Edit and View and Tools...'. Control type 'Menu item' is defined for file and under file controls also defined menu item.
![image](https://user-images.githubusercontent.com/49272759/60995150-9e50f280-a35a-11e9-8099-16c34113689e.png)

**Expected behavior:**
Correct control type only 'Menu' should be define for all menus like 'File and Edit and View and Tools...'.
![image](https://user-images.githubusercontent.com/49272759/60995195-b0cb2c00-a35a-11e9-9273-8491d2d1e111.png)

**Comment by merriemcgaw** [comment]
I will talk with an Accessibility expert on Tuesday. I'll follow up with expected types.

**Comment by vladimir-krestov** [comment]
Currently, the issue of the accessibility navigation tree structure is being resolved.

**Comment by vladimir-krestov** [comment]
The vision of this problem was revised. It was decided to create a separate bug with a modified description of the expected result.
